src/operations/find_and_modify.ts

```diff
@@ -10,6 +10,15 @@ import { Sort, SortForCmd, formatSort } from '../sort';
 import type { ClientSession } from '../sessions';
 import type { WriteConcern, WriteConcernSettings } from '../write_concern';
 
+/** @public */
+export const ReturnDocument = Object.freeze({
```

Comment:
This is completely expected to me given what other drivers do as far as it being defined. I'd just note the options passed to the command are `true` for `AFTER` and `false` for `BEFORE`, so if those aren't the values here they'd need to be mapped somewhere.

Reply:
Oh wait nevermind. I saw to handle that below.